### ISSUE TITLE
Update README.md to provide entry point to the project

### ISSUE THREAD

**Issue by sn0wcat** [amosproj/amos2024ss01-xcelerator-demo-app#40]
## User story

1. As a interested party in this project
2. I want / need to have a good README.md describing the project, linking to the architecture etc.
3. So that I can have a good first experience 

## Acceptance criteria

- [x]   README.md is updated with project logo
- [x]   README.md is updated with short summary of the project
- [x]   README.md is updated with list of participants
- [ ]   README.md is updated with screenshot of the UI
- [x]   README.md is updated with link to Architecture document

## Definition of done (DoD)

- [ ] Added only after week 5
- [ ] The same for all features
- [ ] Here goes the project specific part

## DoD general criteria

- [ ] Feature has been fully implemented
- [ ] Feature has been merged into the mainline
- [ ] All acceptance criteria were met
- [ ] Product owner approved features
- [ ] All tests are passing
- [ ] Developers agreed to release

**Comment by sn0wcat** [comment]
@lamara11 @shahraz1998 can this be please handled with priority? WE should have a presentatable readme, it makes it easier to promote the project :)

**Comment by HaruspexSan** [comment]
@sn0wcat which project logo should i add to the document?
The siemens logo you provided or our own xcelerator logo we made?

**Comment by sn0wcat** [comment]
@HaruspexSan take this one:

![image](https://github.com/amosproj/amos2024ss01-xcelerator-demo-app/assets/1426617/955a9319-afc2-4424-a810-9c35ec9a3ee7)

**Comment by sn0wcat** [comment]
@HaruspexSan i made a few changes, you can merge that and close the issue (after we add the screenshot of the UI)
